### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by NeunEinser**

### CODE REVIEW

File: assets/bingo/lang/pl_pl.json [hunk 1]
```diff
@@ -301,57 +192,40 @@
...
-
 	"bingo.lobby.tutorial.skybox.sign.line1": "Przy starcie,",
 	"bingo.lobby.tutorial.skybox.sign.line2": "masz chwilę",
 	"bingo.lobby.tutorial.skybox.sign.line3": "czasu, aby ",
 	"bingo.lobby.tutorial.skybox.sign.line4": "się rozejrzeć.",
-
-	"bingo.lobby.tutorial.goals.sign.line1": "",
 	"bingo.lobby.tutorial.goals.sign.line2": "Cele",
-	"bingo.lobby.tutorial.goals.sign.line3": "",
-	"bingo.lobby.tutorial.goals.sign.line4": "",
```
Comment (Kri5t0fK) [inline assets/bingo/lang/pl_pl.json:353]: Hmm, it looks like Crowdin's shenanigans.
Those lines shouldn't be deleted.

File: assets/bingo/lang/pl_pl.json [hunk 2]
```diff
@@ -1,12 +1,10 @@
 {
-
-
-
-
```
Comment (Kri5t0fK) [inline assets/bingo/lang/pl_pl.json:5]: Also... those lines got added back (probably also Crowdin)

**Review by Kri5t0fK — CHANGES_REQUESTED** [review]
It looks like Crowdin deletes empty translations 🤔

**Comment by NeunEinser** [comment]
> It looks like Crowdin deletes empty translations 🤔

Yes and no. So I set the setting to make it not push "untranslated" stuff. Untranslated means there is no translation. If you don't set that setting, it puts the original English string in the file by default. So I manually went through all of those and translated them as as space.

So the empty strings that were removed should only be removed in case en_us also uses an empty string.

And if I change source strings (meaning previous en_us empty strings are no longer empty), crowdin should mark them as untranslated again, so that should be fine as well.

File: assets/bingo/lang/pl_pl.json (hunk 2 above)
Comment (NeunEinser) [reply assets/bingo/lang/pl_pl.json:5]: Yeah I marked them hidden on crowdin's side in the meantime, so next sync they should be gone again

File: assets/bingo/lang/pl_pl.json [hunk 3]
```diff
@@ -85,198 +73,103 @@
...
 	"bingo.lobby.card_generation.crafting_table.sign.line2": "Receptur",
 	"bingo.lobby.card_generation.crafting_table.sign.line3": "Rzemieślni-",
 	"bingo.lobby.card_generation.crafting_table.sign.line4": "czych",
-
 	"bingo.lobby.card_generation.furnace.sign.line1": "Wyszukiwarka",
 	"bingo.lobby.card_generation.furnace.sign.line2": "Receptur",
 	"bingo.lobby.card_generation.furnace.sign.line3": "Przetapiania",
-	"bingo.lobby.card_generation.furnace.sign.line4": "",
-
+	"bingo.lobby.card_generation.furnace.sign.line4": " ",
```
Comment (NeunEinser) [inline assets/bingo/lang/pl_pl.json:157]: Here is an example where I needed to manually translate as space to make it work ;)

**Pull request merged**